pkg/config/configuration_test.go

```diff
@@ -17,6 +17,10 @@ const (
 	app1 = "app1"
 	app2 = "app2"
 	app3 = "app3"
+	key1 = "app1_ns1"
```

Comment:
please make the const name explitly. e.g. apps1_namespace1

Reply:
fixed